test/unit/mocked_data/test_get_facts/normal/expected_result.json

```diff
@@ -1 +1 @@
-{"uptime": 79741, "vendor": "Palo Alto Networks", "hostname": "pa1", "fqdn": "N/A", "os_version": "7.0.1", "serial_number": "unknown", "model": "PA-VM", "interface_list": ["ethernet1/1", "ethernet1/2", "ethernet1/3", "ethernet1/4", "loopback"]}
+{"uptime": 1030141, "vendor": "Palo Alto Networks", "hostname": "pa1", "fqdn": "N/A", "os_version": "7.0.1", "serial_number": "unknown", "model": "PA-VM", "interface_list": ["ethernet1/1", "ethernet1/2", "ethernet1/3", "ethernet1/4", "loopback"]}
```

Comment:
Any reason for this change? Just trying to double check I am not missing anything :)